src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -627,15 +625,194 @@
                         *ngIf="placementTargets !== null"
                         [ngValue]="null">-- Select a placement target --</option>
                 <option *ngFor="let placementTarget of placementTargets"
-                        [value]="placementTarget.name">{{ placementTarget.description }}</option>
+                        [value]="placementTarget.name">{{placementTarget.description }}</option>
               </select>
               <cd-help-text>
                 <span i18n>
-                  When creating a bucket, a placement target can be provided as part of the LocationConstraint to override the default placement targets from the user and zonegroup.
+                  When creating a bucket, a placement target can be provided as part of the LocationConstraint to
+                  override the default placement targets from the user and zonegroup.
                 </span>
               </cd-help-text>
             </div>
           </div>
+
+
+
```

Comment:
I don't see this section in bucket-edit form! Is it intended?

Reply:
uploaded video

Reply:
I still do not find that section in the attached video! you can check it by clicking on edit button by selecting any bucket.
Thanks

Reply:
fixed